village/src/components/SmurfForm.js

```diff
@@ -11,15 +13,17 @@ class SmurfForm extends Component {
   }
 
   addSmurf = event => {
-    event.preventDefault();
-    // add code to create the smurf using the api
+  this.postSmurf({
+    'name':`${this.state.name}`,
+    'age':`${this.state.age}`,
+    'height':`${this.state.height}`})
+    }
 
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
-  }
+    postSmurf = smurf =>{
+      axios.post('http:localhost:3333/smurf')
```

Comment:
Remember that a POST request will add data to a server, so you will need to feed it some data to add!